edx/edx-ora2
# Jkantor/invalid default assessments

**TL;DR -** Some of the default template assessments were invalid, this fixes them

JIRA: [EDUCATOR-5560](https://openedx.atlassian.net/browse/EDUCATOR-5560)

**What changed?**

- It's considered invalid to have the `Student Training` step if you don't then also have a `Peer` step
- Many of the default assessments had this issue, leading to confusing behavior (refer to testing instructions)
- Removing the invalid `Student Training` steps will resolve this issue.

**Developer Checklist**


**Testing Instructions**
-Locally, on master, create an ORA and choose one of the following template options: `Self Assessment Only`, `Self Assessment to Staff Assessment`, `Staff Assessment Only`
- Click the "Edit" button
- Without changing anything, immediately click `save`
- You will be given an error message about invalid assessments.

- Install this branch locally and repeat the above steps. There should be no errors.

(Note: `Blank Open Response Assessment` will still raise an error, because there are no steps selected.) 

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:


FYI: @edx/masters-devs-gta
